pkgs/by-name/in/inter-nerdfont/package.nix

```
@@ -0,0 +1,39 @@
+{ lib, pkgs, stdenvNoCC, fetchzip }:
+
+let
+  version = "4.0";
+in stdenvNoCC.mkDerivation {
+  pname = "inter-nerdfont";
+  inherit version;
+
+  src = fetchzip {
+    url = "https://github.com/rsms/inter/releases/download/v${version}/Inter-${version}.zip";
```

Use `finalAttrs` to allow versison overrides
```suggestion
stdenvNoCC.mkDerivation (finalAttrs: {
  pname = "inter-nerdfont";
  version = "4.0";

  src = fetchzip {
    url = "https://github.com/rsms/inter/releases/download/v${finalAttrs.version}/Inter-${finalAttrs.version}.zip";
```

(with extra parenthesis at the end)